client/my-sites/stats/wordads/style.scss

```diff
@@ -0,0 +1,9 @@
+.wordads {
+	.stats-tabs {
+		.stats-tab {
```

Comment:
The nesting is not necessary here and our CSS guidelines [ask us to avoid it](https://github.com/Automattic/wp-calypso/blob/add/wordads-stats/docs/coding-guidelines/css.md#nesting) when possible.

`.wordads .stats-tabs .stats-tab { ... }` is better.